lib/gcp/rules.js

```diff
@@ -20,24 +21,29 @@ function _handleErrorResponse(response) {
 /**
  * Gets the latest ruleset name on the project.
  * @param {String} projectId Project from which you want to get the ruleset.
+ * @param {String} service Service for the ruleset (ex: cloud.firestore or firebase.storage).
  * @returns {String} Name of the latest ruleset.
  */
-function getLatestRulesetName(projectId) {
+function getLatestRulesetName(projectId, service) {
   return api
-    .request("GET", "/" + API_VERSION + "/projects/" + projectId + "/rulesets", {
+    .request("GET", "/" + API_VERSION + "/projects/" + projectId + "/releases", {
       auth: true,
       origin: api.rulesOrigin,
-      data: {
-        pageSize: 1,
-      },
     })
     .then(function(response) {
       if (response.status == 200) {
-        if (response.body.rulesets && response.body.rulesets.length > 0) {
-          return response.body.rulesets[0].name;
+        if (response.body.releases && response.body.releases.length > 0) {
+          var releases = response.body.releases;
+          var prefix = "projects/" + projectId + "/releases/" + service;
+          for (var i = 0; i < releases.length; i++) {
```

Comment:
use lodash find (https://lodash.com/docs/4.17.10#find). Also, what happens if there are multiple rule releases corresponding to firestore? Is that possible? Are all the previous versions kept around? So then do we need to filter for the latest?